docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -10,6 +10,22 @@ A replication factor of at least 3 is highly recommended for any M3DB deployment
 
 M3DB will do its best to distribute shards evenly among the availability zones while still taking each individual node's weight into account, but if some of the availability zones have less available hosts than others then each host in that zone will be responsible for more shards than hosts in the other zones and will thus be subjected to heavier load.
 
+
+### Dangers of Replication Factor of 1 or 2
+
+Running with RF=1 or RF=2 is not recommended for any use cases (testing or production), and in [the future][1107] such
```

Comment:
Not useful for testing even? Assuming I'm building from scratch and running locally (as opposed to e.g. running a docker-compose), I would assume that I'd just run RF=1 for that case. Agreed for every other case of course.

Reply:
I should probably clarify "testing" as in "testing out M3DB". My issue is that it's so unrepresentative of a real-world workload that I'd rather even discourage it for kicking the tires. I've talked to folks who have set up an RF=1 cluster to test on, experienced the slightest blip (like a process being restarted) and the cluster becomes stuck.

Maybe discourage it for testing in a > 1 node cluster?